_plugins/getpub.rb

```diff
@@ -217,7 +217,10 @@ def inspire(pub)
     def load_from_cache(pub, fname)
       return false unless fname.exist?
 
-      f = YAML.load_file fname
+      f = File.open(fname, 'r:bom|utf-8') do |fi|
+        YAML.safe_load(fi, filename: fname, permitted_classes: [Date])
```

Comment:
Ruby 3.0+ has `safe_load_file`. But we are supporting 2.6+ at the moment, so this will do.